samples/Hangfire.Core.MvcApplication/appsettings.json

```diff
@@ -6,7 +6,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection":
-      "Server=(localdb)\\mssqllocaldb;Database=Core_Hangfire_Sample;AttachDBFilename=C:\\Temp\\Sample.mdf;Trusted_Connection=true;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=Core_Hangfire_Sample;AttachDBFilename=C:\\Temp\\Sample.mdf;Trusted_Connection=true;MultipleActiveResultSets=true",
+    "PostgreSqlConnection": "Server=localhost;Port=5432;Database=Hangfire;User Id=postgres;Password=admin;"
```

Comment:
could use Core_Hangfire_Sample to confirm to the mssql convention; it has pw too